t/t2500-job-attach.t

```diff
@@ -129,8 +129,8 @@ test_expect_success 'attach: reports Terminated when tasks are terminated' '
 	grep Terminated terminated.err
 '
 test_expect_success 'attach: reports job shell Killed if job shell is killed' '
-	id=$(flux submit --wait-event=exec.shell.start sh -c "kill -9 \$PPID") &&
-	test_must_fail_or_be_terminated flux job attach $id 2>shell-killed.out &&
+	id=$(flux submit sh -c "kill -9 \$PPID") &&
+	test_must_fail_or_be_terminated flux job attach --wait-event=exec.shell.start $id 2>shell-killed.out &&
```

Comment:
Thanks! I kept meaning to look into this one.

I think the `--wait-event=exec.shell.start` should also be dropped from `flux job attach`. I'm pretty sure it is equivalent to `--wait-event=clean` since I don't think `flux job attach` supports waiting on exec events like `flux submit` does.

In retrospect, the `--wait-event=exec.shell.start` was an accidental cut-and-paste from the previous 2 tests, and should not have been used here I think.

Reply:
Ah that makes sense -thanks!  I pushed an updated patch.